OJS.Workers.Common/Models/ExecutionStrategyType.cs

```diff
@@ -41,6 +41,7 @@ public enum ExecutionStrategyType
         PhpProjectWithDbExecutionStrategy = 36,
         DotNetCoreCompileExecuteAndCheck = 37,
         DotNetCoreUnitTestsExecutionStrategy = 38,
-        SolidityCompileDeployAndRunUnitTestsExecutionStrategy = 39
+        SolidityCompileDeployAndRunUnitTestsExecutionStrategy = 39,
+        DoNothing = 40
```

Comment:
IMO, a comma after the last array element or property is good because, like the line above, in new elements or property is added, them old line won't be marked as changed.